src/utils/components/CloudinitModal/utils/cloudinit-utils.ts

```diff
@@ -129,6 +130,14 @@ export const updateCloudInitRHELSubscription = (
       [ACTIVATION_KEY]: activationKey,
       org: organizationID,
     };
+
+    if (!draftUserDataObject.packages && autoUpdateEnabled) {
```

Comment:
You use this check and the one in the next if statement more than once in separate files. Consider making helper functions with explanatory names that would make these condition expressions more readable.